services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/EndpointServiceImpl.java

```diff
@@ -79,17 +80,21 @@ public EndpointDTO get(String name) {
 	/**
 	 * Create new endpoint object in the System.
 	 * The EndPoint objects should contain Unique values of the 'url' and 'name' fields,
-	 * i.e two objects with same URLs should not be created in the system
+	 * i.e two objects with same URLs should not be created in the system.
 	 * @param userInfo user properties
 	 * @param endpointDTO object with endpoint fields
 	 */
 	@Override
 	public void create(UserInfo userInfo, EndpointDTO endpointDTO) {
-		if(endpointDAO.getEndpointWithUrl(endpointDTO.getUrl()).isPresent()) {
+		//Verify if the Endpoint URL exists in the DataBase
+		Pattern endPointUrl = Pattern.compile(endpointDTO.getUrl(), Pattern.CASE_INSENSITIVE);
```

Comment:
simply endpointUrl

Reply:
I wonder we can move this line to the DAO layer, how do you think?

Reply:
It is a nice idea, becide we cau use the String interface